samples/applets/create-signing-link.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * to run create signing link applet from the project root folder type in your console:
+ * > node samples/applets/create-signing-link <cliend_id> <client_secret> <username> <password> <path_to_file> <fields_stringified>
```

Comment:
```suggestion
 * > node samples/applets/create-signing-link <cliend_id> <client_secret> <username> <password> <document_id>
```

Reply:
lets make the applet simpler. pass only <cliend_id> <client_secret> <username> <password> 
and params required by `createSigningLink` method (<document_id>)